presidio-image-anonymizer/Dockerfile

```
@@ -4,10 +4,15 @@ ARG NAME
 ENV PIPENV_VENV_IN_PROJECT=1
 WORKDIR /usr/bin/${NAME}
 
+RUN apt-get update 
+RUN apt-get install tesseract-ocr -y 
```

Should we fix a specific version? We are fixing `pytesseract` to a specific version. 
@biancafbmd WDYT? How stable is the tesseract API over time?

---

Ideally it should be >5, but I think 4.1.1 is the latest stable one and they mention that modernization of API in 5 cause some issues with compatibility with 4. We are using very basic functionality so think >4 might work as well. More details here: https://tesseract-ocr.github.io/tessdoc/Home.html

---

please notice the versions which are > 5 are still in alpha
the 4.1.1 is the last official release.
I have installed on my machine the v5.0.0-alpha.20201127 and it works perfect.
I don't think it makes a lot of difference 
But we do need to fix it since it seems they are making breaking changes in v5

---

this installs the latest official version which is 4.0.0

---

@omri374 @biancafbmd is this sufficient? or is there a way to spcificly install the 4.1.1 version?

---

Let's keep it as is for now, and experiment with it later. Looks like the API is relatively stable.

---

This should work if we want to fix the version:
```suggestion
RUN apt-get install tesseract-ocr=4.1.1 -y 
```

---

@omri374 this one seems to fail with "Version '4.1.1' for 'tesseract-ocr' was not found" error. I think we will need to clone the repo with the relevant tag and build it in order to get a specific version. 
For now, I will leave the default version as is and we can tackle it later.